test/CMakeLists.txt

```diff
@@ -1528,6 +1528,130 @@ COMMAND ${DYNAMIC_IMPLICITGEMM_XDLOPS_NHWC_WRW_ENVS} $<TARGET_FILE:test_conv2d>
 COMMAND ${DYNAMIC_IMPLICITGEMM_XDLOPS_NHWC_WRW_ENVS} $<TARGET_FILE:test_conv2d> ${MIOPEN_TEST_FLOAT_ARG} --verbose --input  1 128 56 56 --weights 1 128 5 5 --pads_strides_dilations 0 0 2 2 1 1 ${ARGS_NHWC_WRW}
 )
 
+set(DYNAMIC_IMPLICITGEMM_DLOPS_NCHWC_FWD_ENVS
+    ${DYNAMIC_IMPLICITGEMM_COMMON}
+    MIOPEN_DEBUG_FIND_ONLY_SOLVER=ConvAsmImplicitGemmGTCDynamicFwdDlopsNCHWC)
+
+set(ARGS_NCHWC_NCHWC_FWD_FP16x4
+    --cmode convfp16
```

Comment:
What is the intent of "--cmode **_convfp16_**"? Currently `test_conv2d` supports only these values:
https://github.com/ROCmSoftwarePlatform/MIOpen/blob/6f31a5cdbc1cf1c80692d0d1d64d43205ee323c7/test/conv_common.hpp#L1883-L1887

Reply:
it seems miopen decoupled the data type and convolution mode, I'm going to check that